Lib/encodings/aliases.py

```diff
@@ -263,9 +263,12 @@
     'hex'                : 'hex_codec',
 
     # hp_roman8 codec
+    'hp-roman8'          : 'hp_roman8',
```

Comment:
Is it needed?
```
>>> ''.encode('hp-roman8')
b''
```

Reply:
@serhiy-storchaka Shall I drop "hp-roman8" if it is automapped to "hp_roman8"?

Reply:
Yes, please.